ui/app/AppLayouts/Communities/popups/ExportControlNodePopup.qml

```diff
@@ -0,0 +1,147 @@
+            }
+            ColumnLayout {
+                Layout.fillWidth: true
+                spacing: 4
+                Paragraph {
+                    text: qsTr("1. Copy your Community’s private key:")
+                }
+                StatusBaseInput {
+                    id: privateKeyTextArea
+                    Layout.fillWidth: true
+                    multiline: true
+                    edit.readOnly: true
+                    text: root.privateKey
+                    rightComponent: CopyButton {
+                        onClicked: {
+                            privateKeyTextArea.edit.selectAll()
+                            privateKeyTextArea.edit.copy()
+                            privateKeyTextArea.edit.deselect()
+                        }
+                    }
+                }
+                Paragraph {
+                    text: qsTr("2. Stop using this computer as a control node")
+                }
+                Paragraph {
+                    text: qsTr("3. Import this Community via private key on another installation of Status desktop")
+                }
+            }
+            ColumnLayout {
+                Layout.fillWidth: true
+                Layout.fillHeight: true
+                StatusModalDivider { Layout.fillWidth: true }
+                Item { Layout.fillHeight: true }
+                Paragraph {
+                    text: qsTr("I acknowledge that...")
+                }
+
+                StatusCheckBox {
+                    id: agreeToStopControl
+                    Layout.fillWidth: true
+                    font.pixelSize: Style.current.primaryTextFontSize
+                    text: qsTr("%1 will stop working without a control node").arg(root.communityName)
+                }
+                StatusCheckBox {
+                    id: agreeToSavePrivateKey
+                    Layout.fillWidth: true
+                    Layout.minimumHeight: 40
+                    font.pixelSize: Style.current.primaryTextFontSize
+                    text: qsTr("I have saved the %1 private key").arg(root.communityName)
+                }
+                StatusCheckBox {
+                    id: agreeToDeletePrivateKey
+                    Layout.fillWidth: true
+                    Layout.minimumHeight: 40
+                    font.pixelSize: Style.current.primaryTextFontSize
+                    text: qsTr("If I lose the private key, %1 will be unrecoverable").arg(root.communityName)
+                }
+            }
+        }
+    }   
+    rightButtons: [
+        StatusButton {
+            text: qsTr("Delete private key and stop control node")
+            enabled: agreeToStopControl.checked && agreeToSavePrivateKey.checked && agreeToDeletePrivateKey.checked
+            type: StatusBaseButton.Type.Danger
+            onClicked: {
+                root.deletePrivateKey()
```

Comment:
[Here](https://github.com/status-im/status-desktop/issues/11572#issuecomment-1641808074) is the discussion we are (probably) going to call privateKey by a signal from a new control node

Reply:
This action is not needed if we're going to implement the signal from a new control node. I'd say to go forward with the UI in this form and update it as needed once we have the backend implementation.

Thanks for letting me know!